### PR TITLE
fix(actions): remove more option button AP-419

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Removed more option button

**Which issue(s) this PR fixes** 🔨

AP-419

### CODE REVIEW

File: components/views/chat/message/actions/Actions.html [hunk 1]
```diff
@@ -31,10 +31,10 @@
...
     <edit-icon size="1x" :class="'control-icon'" />
   </div>
-  <div
+  <!-- <div
     class="reply-command has-tooltip has-tooltip-primary"
     :data-tooltip="$t('ui.more')"
   >
     <more-vertical-icon size="1x" :class="'control-icon'" />
-  </div>
+  </div> -->
```
Comment (Jekrimo) [inline components/views/chat/message/actions/Actions.html:39]: Can we just delete this whole commented code? Or are we planning to uncomment for later use?

**Review by Jekrimo — CHANGES_REQUESTED** [review]

File: components/views/chat/message/actions/Actions.html (hunk 1 above)
Comment (davidchan1219) [reply components/views/chat/message/actions/Actions.html]: @Jekrimo I removed that code.

**Comment by stavares843** [comment]
i don't think the idea was to remove the button

i think the idea was something more

if there are options show the button

if not, hide the button

cc @WanderingHogan thoughts?

**Comment by WanderingHogan** [comment]
@davidchan1219 like @stavares843 said - this isn't meant to remove the ellipses, just hide them if there are not things to show in the ellipses, more likely a v-if

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
hide if nothing in it, show if something in it, not remove code

**Comment by davidchan1219** [comment]
@WanderingHogan There is no any item in the settings, so I made to hide the more icon.

**Comment by stavares843** [comment]
@davidchan1219 feel free to remove changes requested label and add missing code review label when you address the feedback

**Comment by stavares843** [comment]
and thanks for updating 🔨

**Review by phillsatellite — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**